profiling-ffi/src/crashtracker/crash_info.rs

```diff
@@ -0,0 +1,214 @@
+// Copyright 2021-Present Datadog, Inc. https://www.datadoghq.com/
+// SPDX-License-Identifier: Apache-2.0
+
+use super::{CrashtrackerConfiguration, CrashtrackerMetadata, SigInfo};
+use crate::crashtracker::{
+    crashinfo_ptr_to_inner, option_from_char_slice, CrashInfo, CrashInfoNewResult,
+    CrashtrackerResult, StackFrame,
+};
+use crate::exporter::{self, ProfilingEndpoint};
+use anyhow::Context;
+use ddcommon_ffi::{slice::AsBytes, CharSlice, Slice};
+use std::time::Duration;
+
+/// Create a new crashinfo, and returns an opaque reference to it.
+/// # Safety
+/// No safety issues.
+#[no_mangle]
+#[must_use]
+pub unsafe extern "C" fn ddog_crashinfo_new() -> CrashInfoNewResult {
```

Comment:
Should this be `ddog_CrashInfo_new`? (And a similar change for the other functions in this file?)